play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/hmrcfrontend/config/RebrandConfig.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.config
+
+import play.api.Configuration
+
+import javax.inject.Inject
+
+case class RebrandConfig @Inject() (config: Configuration) {
+  val useRebrand: Boolean = config.getOptional[Boolean]("play-frontend-hmrc.useRebrand").getOrElse(false)
```

Comment:
I think we need a comment that highlights that this is slightly different to how the feature flag in govuk-frontend works (where their approach still allows it to be overridden through component args, but ours forces the rebrand - so people should only use ours if they don't need to conditionally rebrand some pages)

Reply:
Added comment, please take a look if you are happy with the content :)